static/styles/left_sidebar.css

```diff
@@ -464,6 +464,33 @@ li.expanded_private_message {
         position: relative;
         top: 0;
     }
+
+    .group_size {
+        padding: 0 4px;
+        background-color: inherit;
+        margin-left: -1px;
+        color: inherit;
+        border: 0.5px solid hsl(105, 2%, 50%);
+        height: 16px;
+        line-height: 16px;
+        font-size: 12px;
+        font-weight: normal;
+        letter-spacing: 0.6px;
+        border-radius: 4px;
+    }
+
+    /*
+    Here large_group_size stands for the group where users
+    count are greater than 9.
+*/
+    .large_group_size {
+        /*
+        Shifting the .group_size more towards left side to
+        avoid overlapping of the text and count when the group
+        contain more than 9 uers.
+    */
+        margin-left: -5px;
```

Comment:
I think we can do this correctly using `float: right`, like we do with the unread counts, and get to eliminate the `large_group_size` CSS class.